SMLHelper/Assets/ModPrefab.cs

```diff
@@ -81,28 +81,25 @@ internal GameObject GetGameObjectInternal()
             go.transform.position = new Vector3(-5000, -5000, -5000);
             go.name = this.ClassID;
 
-            /* Make sure prefab doesn't get cleared when quiting game to menu. */
-            SceneCleanerPreserve scp = go.AddComponent<SceneCleanerPreserve>();
-            scp.enabled = true;
-
             if (this.TechType != TechType.None)
             {
-                go.AddComponent<Fixer>().techType = this.TechType;
+                if (go.activeInHierarchy) // don't add Fixer to inactive prefabs
+                    go.AddComponent<Fixer>();
```

Comment:
If prefab is not active (e.g. it's loaded from asset bundle) then `Fixer` will not destroy it (because it'll not updated) and next `Fixers` will be added to the same prefab (e.g. BaseClocks mod works that way).

Reply:
I understand the first part but why remove the assigning of the techtype?

Reply:
Maybe I'm wrong here, but I think techtype was needed when Fixer used ProtoSerialize methods. Prefabs already have `TechTag` and `Constructable` with techtypes and techtype in Fixer is not used.

Reply:
If they're no longer needed, for the same reason that ProtoSerialize isn't needed anymore, I see no reason not to simplify the code.